Apps/WebClient/src/Server/Services/WalletStatusService.cs

```diff
@@ -102,11 +102,13 @@ public async Task<RequestResult<WalletCredential>> UpdateIssuedCredential(Webhoo
             if (dbResult.Status == DBStatusCode.Read)
             {
                 WalletCredential credential = dbResult.Payload;
+                WalletCredentialStatus readStatus = credential.Status;
                 retVal.ResourcePayload = credential;
+                credential.Status = WalletCredentialStatus.Added;
```

Comment:
Because of this, we're now setting the Status to Added twice.  I think line 120 can be removed.